examples/container-with-vitest/test/ReactWrapper.test.ts

```diff
@@ -0,0 +1,20 @@
+import { experimental_AstroContainer as AstroContainer } from 'astro/container';
+import { expect, test } from 'vitest';
+// @ts-ignore
+import ReactWrapper from '../src/components/ReactWrapper.astro';
+
+test('ReactWrapper with react renderer', async () => {
+	const container = await AstroContainer.create({
+		renderers: [
+			{
+				name: '@astrojs/react',
+				clientEntrypoint: "@astrojs/react/client.js",
+				serverEntrypoint: "@astrojs/react/server.js",
+			}
```

Comment:
For the future, but will we be providing an easier way to build this for users?

Reply:
Definitely. At the moment no integration exposes something like that :(